frontend/apps/pablo/components/Atoms/AreaChart.tsx

```diff
@@ -127,8 +127,12 @@ export const AreaChart: React.FC<AreaChartProps> = ({
 }) => {
   const theme = useTheme();
 
-  let min = Math.min(...data.map((x) => x[1]));
-  let max = Math.max(...data.map((x) => x[1]));
+  let { min, max } = useMemo(() => {
+    let min = data.reduce((agg, [ts, val]) => (val < agg ? val : agg), Infinity)
+    let max = data.reduce((agg, [ts, val]) => (val > agg ? val : agg), -Infinity)
```

Comment:
Infinity doesn't sound like a good fallback value, since you are using min, max in arithmetic logic further down. 

Possible to replace them with safer values to prevent `NaN` like `0`?